Готовая 16 домашка.

baldezhnik opened the pull request:
Сразу вопрос, если у Вас уже выбраны следующие задания для дз17, можно их заранее получить, т.к. выходные будут загружен. Буду очень благодарен.

baldezhnik committed:
Создана ветка для выполнения HW16

baldezhnik committed:
накидан макет layout, сделаны домашки 1,3,5,6,7

baldezhnik committed:
доделано активити, допилены все домашки кроме 2

baldezhnik committed:
я не знаю понял ли я 2 задние, но сделал как понял его

OKurcheuskaya commented:
dz 17 https://play.kotlinlang.org/koans/overview (прислать скрин прогресса)

обратный отсчет (таймер)

https://medium.com/android-dev-hacks/introduction-to-kotlin-coroutines-72a951577468  возможно еще что-то будет

OKurcheuskaya commented on app/src/main/java/com/dsd/homework/HW16Activity.kt:
```diff
@@ -27,45 +27,81 @@ class HW16Activity : AppCompatActivity() {
+            val a2 = (0..20).random()
+            val b2 = (0..20).random()
+            val str: String
+            if (((a1 > a2) and (b1 > b2)) or ((a1 > b2) and (b1 > a2))) str =
```
перепиши на when

OKurcheuskaya commented on app/src/main/res/layout/activity_hw16.xml:
```diff
@@ -8,8 +8,8 @@
   <TextView
-      android:id="@+id/textView30"
-      android:layout_width="wrap_content"
+      android:id="@+id/txtv_act16_hw4"
```
название поменяй на tv_act16_hw4

baldezhnik committed:
Изменения согласно замечаниям
изменены переменные и индексы TextView
изменена функция в 4 задании с if на when
